Vagrantfile

```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+	config.vm.box = "centos/7"
+	config.vm.synced_folder ".", "/vagrant", disabled: true
+	config.vm.synced_folder "./samples", "/converge_samples"
+	config.vm.provision :converge do |cvg|
```

Comment:
One of the things we need for the full-integration test is to check to make sure we have the latest version of the plugin. I haven't written that script yet